src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -147,28 +147,11 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     logger.LogTempKey(options.TemporaryJwtKey.Token);
                 }
-                //Auth is enabled and we are binding on http. Make sure we log a warning.
 
-                string hostingUrl = Configuration.GetValue<string>(WebHostDefaults.ServerUrlsKey);
```

Comment:
This refactor is necessary due to the value being cleared from configuration by the point this executes.